src/api/app/views/webui2/webui/package/live_build_log.html.haml

```diff
@@ -35,5 +35,8 @@
       = render partial: 'live_build_log_controls'
 
 :javascript
+  faviconFinished = '#{path_to_image('favicon-warning.ico')}'
+  faviconRunning = '#{path_to_image('favicon-info.ico')}'
+  faviconPaused = '#{path_to_image('favicon.ico')}'
```

Comment:
How about passing all those paths as new parameters in the LiveLog constructor below? That way, there's no need to define global variables and the jslinter will not complain.

Please, remember to make the same change in the previous `live_build_log.html.erb`.

Reply:
Good point, sounds saner than the current way :smile:

Reply:
@hellcp do you think you find the time to go over it?